### PR TITLE
Added sentry JS support.

### PR DESCRIPTION
**Pull request by jayelkaake**
@benjolabs @lwilkowskeBC

This adds JS sentry support to the sentry module.

### CODE REVIEW

**Comment by jayelkaake** [comment]
ping @benjolabs @lwilkowskeBC

File: SentryPlugin.php [hunk 1]
```diff
@@ -112,5 +129,50 @@ public function init()
...
+
+        if (!$this->isWebRequest()) {
+            return $this;
+        }
+
+        if (!$settings->reportJsErrors) {
+            return $this;
+        }
+
+        craft()->templates->includeJsFile('https://cdn.ravenjs.com/1.1.22/jquery,native/raven.min.js');
```
Comment (ghost) [inline SentryPlugin.php:154]: @benjolabs : is including what looks like another version of jQuery going to mess with any of our FE stuff?

Comment (jayelkaake) [inline SentryPlugin.php:154]: I don't actually think it is including jquery. Yeah it looks that way
though until I checked what code was actually included and it was not.​

Comment (benjolabs) [inline SentryPlugin.php:154]: yeah looks like it's just if you want to use jQuery.
Is there anything we need add to the FE code / templates?

Comment (jayelkaake) [inline SentryPlugin.php:154]: No, nothing we need to add to frontend templates.

**Comment by jayelkaake** [comment]
:recycle: @benjolabs @lwilkowskeBC

**Comment by ghost** [comment]
We determined that this wasn't loading up a bunch of duplicate libraries right?

If it's not, then :+1:  Lets give it a shot.

**Comment by jayelkaake** [comment]
Yeah I confirmed. Thanks!

Jay

**Pull request merged**